@nav-frontend/react-link/src/Link.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { forwardRef } from "react";
+import cl from "classnames";
+import "@nav-frontend/link-styles";
+
+export interface LinkProps
+  extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
+  /**
+   * Link anchor should direct to
+   */
+  href: string;
```

Comment:
Skal href være required?

Reply:
Er ikke required for default `<a>`, men lintere gir vanligvis beskjed hvis du ikke har en href. Siden lintere ikke gjør dette med custom `<Link>` er den satt som required. Vi kan ta et valg her å ikke "impose" riktig bruk på bruker, men heller dokumentere det istedenfor?